e2e/specs/create.task.spec.js

```diff
@@ -20,4 +22,6 @@ describe('pivotal tracker project page add task to story', () => {
         storyTasks.setTaskTitleTextField('new task');
         storyTasks.clickSaveAndAddNewTaskButton();
     });
+
+    //TODO after delete project from api
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)